data/tooltips.txt

```diff
@@ -840,7 +843,10 @@ tip "% shield damage / shot:"
 	`Each shot fired by this weapon does this amount of shield damage as a percentage of the target ship's total shields.`
 
 tip "% hull damage / shot:"
-	`Each shot fired by this weapon does this amount of hull damage as a percentage of the target ship's total hull.`
+	`Each shot fired by this weapon does this amount of hull damage as a percentage of the target ship or asteroid's total hull.`
+
+tip "% asteroid damage / shot:"
+	`Each shot fired by this weapon does this amount of asteroid damage as a percentage of the target asteroid's total health, in lieu of relative hull damage being used.`
```

Comment:
```suggestion
	`Each shot fired by this weapon does this amount of damage as a percentage of the target asteroid's total health, in lieu of relative hull damage being used.`
```

Sounds better idk. What do you think?

Reply:
Yh, sounds much better, I'll commit the suggestion.